src/docs/issue-platform-detectors.mdx

```diff
@@ -127,3 +127,26 @@ default={
     [Project settings:](https://sentry.io/settings/sentry/projects/sentry/performance/)
 
     ![Project settings screenshot](img/issuePlatformFingerprinting03.png)
+
+## Detector Threshold Configurations
+
+Once the detector has been tested and deployed, you can add the corresponding issue to the list of detector threshold settings in **Project Settings > Performance**:
+
+![Detector Threshold settings customer ui screenshot](img/issuePlatformCustomerDetectorSettings.png)
+
+Also, add the corresponding issue to the admin settings ui. It allows support to enable/disable detection of the issue for a project:
+
+![Detector Threshold settings admin ui screenshot](img/issuePlatformAdminDetectionSettings.png)
+
+Things to note when making detector thresholds configurable: 
+
+- The **maximum threshold value** of the range, should not be big enough to make the issue undetectable. We are not trying to disable detection here. 
+- The **minimum threshold value** of the range, should hold some significance when exceeded. We want our users to find a issue that we detect, significant enough to look further into. 
+- The **impact** of changing the thresholds chosen to be made configurable, **should be easy for the user to interpret**. For example: Instead of displaying the raw duration of db base spans on the main thread
+ as a configurable threshold, we display the frame rate drop that may result from having a db span of that duration on the main thread:
```

Comment:
```suggestion
- The **impact** of changing the configurable thresholds **should be easy for the user to interpret**. For example, instead of configuring the raw duration of DB base spans on the main thread, users can configure the resulting frame rate drop. 
```